Management/src/ActuatorWeb/Program.cs

```diff
@@ -0,0 +1,107 @@
+using Steeltoe.Common;
+using Steeltoe.Configuration.CloudFoundry;
+using Steeltoe.Management.Endpoint;
+using Steeltoe.Management.Endpoint.Actuators.All;
+using Steeltoe.Management.Endpoint.SpringBootAdminClient;
+using Steeltoe.Management.Prometheus;
+using Steeltoe.Management.Tracing;
+using Steeltoe.Samples.ActuatorWeb;
+using Steeltoe.Samples.ActuatorWeb.CustomActuators.LocalTime;
+
+WebApplicationBuilder builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+builder.Services.AddRazorPages();
+
+// Steeltoe: Add Cloud Foundry configuration provider and options classes.
+builder.AddCloudFoundryConfiguration();
+
+// Steeltoe: Add all actuators.
+builder.Services.AddAllActuators();
+
+// Steeltoe: Add Prometheus and specify an authorization policy for use outside Cloud Foundry.
+if (!Platform.IsCloudFoundry)
+{
+    builder.Services.ConfigureActuatorAuth();
+
+    builder.Services.ConfigureActuatorEndpoints(configureEndpoints =>
+    {
+        configureEndpoints.RequireAuthorization("actuator.read");
+    });
+
+    // Steeltoe: Map Steeltoe-configured OpenTelemetry Prometheus exporter with authorization middleware in the pipeline.
+    builder.Services.AddPrometheusActuator(true, configurePrometheusPipeline =>
+    {
+        configurePrometheusPipeline.UseAuthorization();
+    });
+}
+else
+{
+    // Steeltoe: Map Steeltoe-configured OpenTelemetry Prometheus exporter.
+    builder.Services.AddPrometheusActuator();
+}
+
+// Steeltoe: Add custom actuator that displays the local server time.
+// See appsettings.Development.json for how the response format can be configured.
+builder.Services.AddLocalTimeActuator();
+
+// Steeltoe: Register with Spring Boot Admin.
+if (builder.Configuration.GetValue<bool>("UseSpringBootAdmin"))
+{
+    builder.Services.AddSpringBootAdminClient();
+}
+
+// Steeltoe: Ensure log entries include "[app-name, traceId, spanId]" for log correlation.
+builder.Services.AddTracingLogProcessor();
+
+// Steeltoe: Configure OpenTelemetry app instrumentation and export of metrics and traces.
+builder.Services.ConfigureOpenTelemetry(builder.Configuration);
+
+// Steeltoe: Register typed HttpClient for communicating with the backend service.
+builder.Services.AddHttpClient<ActuatorApiClient>(SetBaseAddress);
+
+WebApplication app = builder.Build();
+
+// Configure the HTTP request pipeline.
+if (!app.Environment.IsDevelopment())
+{
+    app.UseExceptionHandler("/Error");
+    // The default HSTS value is 30 days. You may want to change this for production scenarios, see https://aka.ms/aspnetcore-hsts.
+    app.UseHsts();
+}
+
+// Steeltoe: See the note in the README under "Regarding HTTPS and Basic Authentication"
+// app.UseHttpsRedirection();
```

Comment:
I don't consider this change an improvement, because it no longer explains why auto-redirect must be turned off.
Suggestion:
> // Steeltoe: The next line is commented out because Actuators are listening on a dedicated non-https port.